elastic/test/test_elastic.py

```diff
@@ -315,7 +315,7 @@
 def get_es_version():
     version = os.environ.get("FLAVOR_VERSION")
     if version is None:
-        return [1, 6, 0]
+        return [1, 3, 9]
```

Comment:
Any reason for this change? Not that there's anything wrong with it specifically, just curious.

Reply:
Tests were failing by default because there was a version mismatch with the rake file :
https://github.com/DataDog/integrations-core/blob/7d94fb5c546bcdc90ed6f934049856bccda5bdce/elastic/test/ci/elastic.rake#L3-L5